src/components/OutlinedCardList.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '10px',
+      border: '1px solid',
+      borderColor: theme.palette.divider,
+      borderRadius: '6px',
+    },
+    content: {
+      padding: '8px 12px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    itemWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    items: {
+      width: '100px',
+      color: theme.palette.text.primary,
+      margin: '5px',
+    },
+  }),
+);
+
+interface CardProps {
+  numbers: number[];
+  labels: string[];
+  unit: string[];
+  primary: boolean;
+  rightIcon?: JSX.Element;
+}
+
+export default function OutlinedColCard(props: CardProps) {
```

Comment:
can we keep the name of component and the filename the component lives in in sync?

Reply:
Also a bit of a NIT but if it's not a lot of trouble I think it's easier to keep abbreviations out of component names if possible. `OutlinedColumnCard` if `Col` stands for Column (but feel free to ignore if you feel strongly the other way)